pkg/apis/kubernetes/v1/unit_test.go

```diff
@@ -0,0 +1,28 @@
+//go:build !integration
+// +build !integration
+
+package v1
+
+import (
+	"context"
+
+	"go.anx.io/go-anxcloud/pkg/api/types"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const isIntegrationTest = false
+
+var _ = Describe("Unit Tests", Ordered, func() {
```

Comment:
please don't `Describe("Unit Tests")`, in this case it's probably `Describe("NodePool resource") / It("can filter by cluster")`